addons/binding/org.openhab.binding.kodi/README.md

```diff
@@ -105,16 +105,16 @@ sitemap demo label="myKodi"
     Frame label="myKodi" {
         Switch    item=myKodi_mute
         Slider    item=myKodi_volume
-        Selection item=myKodi_control mappings=[PLAY='Play', PAUSE='Pause', NEXT='Next', PREVIOUSE'Previous', FFWARD='Fastforward', REWIND='Rewind']
+        Selection item=myKodi_control mappings=[PLAY='Play', PAUSE='Pause', NEXT='Next', PREVIOUSE'Previous', FASTFORWARD='Fastforward', REWIND='Rewind']
+        Default   item=myKodi_control
         Switch    item=myKodi_stop
         Text      item=myKodi_title
         Text      item=myKodi_showtitle
         Text      item=myKodi_album
         Text      item=myKodi_artist
         Text      item=myKodi_playuri
         Text      item=myKodi_notification
```

Comment:
You removed the `inputtext` element in the sitemap because there is no element for inserting text implemented, yet. So why don't you remove the `notification` element, too?